build.sh

```diff
@@ -140,7 +140,7 @@ pushd php/php-k8s-v81-NR && \
 
 # Build the php 8.1 builder image.
 pushd php/builder81 && \
-  make VERSION=${VERSION81} EXTRAVERSION=${EXTRAVERSION} UPSTREAM=16-alpine build && \
+  make VERSION=${VERSION81} EXTRAVERSION=${EXTRAVERSION} UPSTREAM=16.19.0-alpine build && \
```

Comment:
is this the latest alpine (3.17) and the older builder just needs 3.16 or is it a typo?

Reply:
This is the latest 3.17, which only has PHP8.1. The PHP8.0 needs the older Alpine 3.16.